src/components/ebay-listbox-button/README.md

```diff
@@ -34,6 +34,7 @@ Name | Type | Stateful | Required | Description
 `selected` | Number | Yes | n/a | allows you to set the selected index option to `selected`
 `borderless` | Boolean | No | No | whether button has borders
 `fluid` | Boolean | No | No | whether listbox width is 100%
+`buttonName` | String | No | Yes | used for the `name` attribute of the native `<button>`
```

Comment:
@agliga @seangates Is this correct? `buttonName` instead of just `name`? I thought this would just be a pass through.

Reply:
`name` is what it should be. And, we probably don't need to document it. We simply should handle it and place the attribute in the right place in the template.

So, the only changes should be:
- add the attribute to `marko.json`
- add the attribute to button in the template

Maybe we could even do without the tests? I can see that being good to know the name was set on the button instead of the wrapper, though.

Reply:
@seangates `name` is already in use:
https://github.com/eBay/ebayui-core/blob/56b70df668de30d113de683b7cb86e5f72a4fe9c/src/components/ebay-listbox-button/template.marko#L65.

Do you think we should use the same option at both the places?

Reply:
So from what it looks like is we probably have to move the name from the select to the button. 
Since the select listbox button has the value of the selected options, when submitting the form `button` will get those values and map it to name. 
If we have `select` have the same name, it will conflict and have two of the same keys. 
That or we have another option for name/selectName. Also moving name to button/changing that functionality might be breaking.

Reply:
Ah, good point @agliga . Given that the purpose of the listbox-button is to submit form data, then the server should be getting the key/value pairs from the name of the select element. Not entirely sure why they want to query the button instead, but I think the PR as it currently stands should satisfy that and also be backwards compatible...?